source/extensions/quic_listeners/quiche/udp_gso_batch_writer_config.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/network/udp_packet_writer_config.h"
+#include "envoy/registry/registry.h"
+
+namespace Envoy {
+namespace Quic {
+
+const std::string GsoBatchWriterName{"udp_gso_batch_writer"};
+
+// UdpPacketWriterConfigFactory to create UdpGsoBatchWriterFactory based on given protobuf.
+class UdpGsoBatchWriterConfigFactory : public Network::UdpPacketWriterConfigFactory {
+public:
+  ProtobufTypes::MessagePtr createEmptyConfigProto() override;
```

Comment:
nit: // Network::UdpPacketWriterConfigFactory

Reply:
Updated the comment!
```
// Network::UdpPacketWriterConfigFactory to create UdpGsoBatchWriterFactory based on given 
// protobuf.
```